plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/logcat/LogcatLevel.kt

```diff
@@ -0,0 +1,25 @@
+package io.sentry.android.gradle.instrumentation.logcat
+
+enum class LogcatLevel {
+    VERBOSE,
+    DEBUG,
+    INFO,
+    WARNING,
+    ERROR;
+
+    fun allowedLogFunctions(): List<String> {
```

Comment:
This function will probably get called many times, so it could make sense to optimize this avoid the list allocations. How about assigning a number to every level and then just comparing those? Something like: 
````
enum class LogcatLevel(val level : Int) {
    Verbose(0),
    Debug(1);

    fun supports(other: LogcatLevel) : Boolean {
        return level >= other.level
    }
}
````